### DISCUSSION TITLE
Badges for matrix builds

### DISCUSSION THREAD

**Discussion by patrikhuber** [actions/virtual-environments#7413] [Ideas]
Hi,

I'm using GitHub Actions to CI-build a software on Linux, Windows and macOS. I'm using a build matrix for that. However using the current badges, only a blanked "success" or "failing" is displayed. I'd like to display separate badges for the Linux, Windows and macOS builds.

I was able to find one or two third-party websites for this purpose but neither of them looked like it would still be here in a couple years' time (e.g. https://github.com/40ants/github-actions) and I'd like something (ideally official) that I can trust that it will keep working.

Thanks!

**Comment by mikhailkoliada** [comment]
Hello! Please ask in https://github.com/orgs/community/discussions/

**Reply by patrikhuber** [reply]
Hi, I've created https://github.com/orgs/community/discussions/52616.
